OpenRA.Game/Widgets/Widget.cs

```diff
@@ -110,6 +110,14 @@ public static bool HandleInput(MouseInput mi)
 
 		public static bool HandleKeyPress(KeyInput e)
 		{
+			if (Hotkey.FromKeyInput(e) == Game.Settings.Keys.TakeScreenshotKey)
```

Comment:
RootWidget was created to make these global hotkey hacks a bit more tolerable :)